### PR TITLE
Make MRI a separate acquisition session and removed from procedures

### PR DESCRIPTION
**Pull request by saskiad**
address issue #198

### CODE REVIEW

File: src/aind_data_schema/ephys/mri_session.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    
+
+class MagneticStrength(Enum):
+    """Strength of magnet"""
+
+    MRI_7T = 7
+    MRI_14T = 14
+
+
+class MriScan(AindModel):
```
Comment (dyf) [inline src/aind_data_schema/ephys/mri_session.py:33]: call this `MriSession` to match the file name. also, make it inherit from `AindCoreModel`. Then delete `describedBy` - it comes for free when you inherit from `AindCoreModel`.

File: src/aind_data_schema/ephys/mri_session.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    )
+    animal_weight_post: Optional[float] = Field(
+        None,
+        title="Animal weight (g)",
+        description="Animal weight after procedure",
+        units="g",
+    )
+    weight_unit: WeightUnit = Field(WeightUnit.G, title="Weight unit")
+    anaesthesia: Optional[Anaesthetic] = Field(None, title="Anaesthesia")
+    scan_sequence: MriScanSequence = Field(..., title="Scan sequence")
```
Comment (dyf) [inline src/aind_data_schema/ephys/mri_session.py:71]: add a `scanner` Device here?

**Review by dyf — CHANGES_REQUESTED** [review]

File: src/aind_data_schema/ephys/mri_session.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    mri_scanner: Scanner = Field(..., title="MRI scanner")
+    resolution: float = Field(..., title="Resolution")
```
Comment (dyf) [inline src/aind_data_schema/ephys/mri_session.py:76]: add `resolution_unit`, presumably mm.

File: src/aind_data_schema/ephys/mri_session.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+class MriSession(AindCoreModel):
+    """Description of an MRI scan"""
+
+    schema_version: str = Field("0.0.1", description="schema version", title="Version", const=True)
+    subject_id: str = Field(
+        ...,
+        description="Unique identifier for the subject. If this is not a Allen LAS ID, indicate this in the Notes.",
+        title="Subject ID",
+    )
+    start_date: date = Field(..., title="Start date")
```
Comment (dyf) [inline src/aind_data_schema/ephys/mri_session.py:51]: should these be `session_start_time` and `session_end_time` to match `ephys_session`?

**Review by dyf — CHANGES_REQUESTED** [review]

File: src/aind_data_schema/ephys/mri_session.py (hunk 3 above)
Comment (saskiad) [reply src/aind_data_schema/ephys/mri_session.py]: added axes from imaging acquisition that has the unit included

**Review by dyf — APPROVED** [review]

**Pull request merged**